tests/cmdline/commands/test_computer.py

```diff
@@ -772,76 +772,75 @@ def test_computer_relabel(self):
         # The new label should be available
         orm.Computer.collection.get(label='comp_cli_test_computer')
 
-    def test_computer_delete_with_nodes(self):
-        """check if 'verdi computer delete' works when there are associated nodes"""
+    # ensure that 'yes' works for backwards compatibility, see issue #6422
+    @pytest.mark.parametrize('user_input', ['y', 'yes'])
+    def test_computer_delete_existing_computer_successful(self, user_input):
+        """Test if `verdi computer delete` works correctly for an existing computer when it is deleted"""
         from aiida.common.exceptions import NotExistent
 
-        label = 'computer_69'
-        compute_temp = orm.Computer(
+        label = 'computer_temp'
+        computer_temp = orm.Computer(
             label=label,
             hostname='localhost',
             transport_type='core.local',
             scheduler_type='core.direct',
             workdir='/tmp/aiida',
         )
-        compute_temp.store()
-        compute_temp.configure(safe_interval=0)
+        computer_temp.store()
+        computer_temp.configure(safe_interval=0)
 
-        c_label = 'code_69'
+        code_label = 'code_temp'
         orm.InstalledCode(
-            label=c_label,
+            label=code_label,
             default_calc_job_plugin='core.arithmetic.add',
-            computer=compute_temp,
+            computer=computer_temp,
             filepath_executable='/remote/abs/path',
         ).store()
 
-        false_user_input = 'y'  # most common mistake
-        user_input = 'yes'
-
-        # Abort in case of wrong input
-        self.cli_runner(computer_delete, [label], user_input=false_user_input, raises=True)
-        orm.load_code(c_label)
-
-        # Safety check in case of --dry-run
-        options = [label, '--dry-run']
-        self.cli_runner(computer_delete, options)
-        orm.load_code(c_label)
-
         # A successul delete, including all associated nodes
         self.cli_runner(computer_delete, [label], user_input=user_input)
 
         with pytest.raises(NotExistent):
             orm.Computer.collection.get(label=label)
 
         with pytest.raises(NotExistent):
-            orm.load_code(c_label)
-
-    def test_computer_delete(self):
-        """Test if 'verdi computer delete' command works"""
-        from aiida.common.exceptions import NotExistent
+            orm.load_code(code_label)
 
-        # Setup a computer to delete during the test
-        label = 'computer_for_test_label'
-        orm.Computer(
+    def test_computer_delete_existing_computer(self):
```

Comment:
I found this test name kinda misleading for it's purpose
```suggestion
    def test_computer_do_not_delete(self):
```

Reply:
named it `test_computer_delete_existing_computer_not_deleted` do be consistent with the test name of the other one